core/src/main/resources/hudson/node_monitors/LabelsMonitor/column.jelly

```diff
@@ -0,0 +1,54 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:s="/lib/form">
+  <style type="text/css">
```

Comment:
This styling is inserted for each node. Where would this need to go to to avoid this?